src/games/strategy/triplea/ai/proAI/util/ProUtils.java

```diff
@@ -46,6 +47,26 @@ public ProUtils(final ProAI ai) {
     return unitTerritoryMap;
   }
 
+  public List<PlayerID> getEnemyPlayersInTurnOrder(final PlayerID player) {
```

Comment:
IMO you're describing the problem that dependency injection is meant to be solve. I think it's time we start incorporating Dagger (we had discussed IOC containers earlier, Dagger was a stronger candidate than others, such as Spring for example)